javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -944,12 +959,16 @@ void PrintProtoDtsFile(Printer* printer, const FileDescriptor* file) {
 void PrintFileHeader(Printer* printer, const std::map<string, string>& vars) {
   printer->Print(
       vars,
+      "// Code generated by protoc-gen-grpc-web. DO NOT EDIT.\n"
+      "// versions:\n"
+      "// \tprotoc-gen-grpc-web v$version$\n"
+      "// \tprotoc              v$protoc_version$\n"
+      "// source: $source_file$\n"
       "/**\n"
```

Comment:
Not exactly sure if closure compiler expects this block to be at the top of the file bug hopefully this is ok :)

Reply:
Ahh from the test failures it seems like you maybe cannot move this section above the @fileoverview.. :)